ci/build.ps1

```diff
@@ -1,4 +1,5 @@
-Set-PSDebug -Trace 1
+$ErrorActionPreference = 'stop'
```

Comment:
This caught the cmake error for mingw build but not MSVC build.
When does `'Running functional tests failed with error'` appear in a failed MSVC build for nvim functional tests?